modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -1764,23 +1768,32 @@ private CompletableFuture<WriteIntentSwitchReplicatedInfo> applyWriteIntentSwitc
             UUID transactionId,
             boolean commit,
             HybridTimestamp commitTimestamp,
+            Set<Integer> tableIds,
             int catalogVersion
     ) {
+        storageUpdateHandler.switchWriteIntents(
+                transactionId,
+                commit,
+                commitTimestamp,
+                indexIdsAtRwTxBeginTsOrNull(transactionId)
+        );
+
+        WriteIntentSwitchReplicatedInfo result = new WriteIntentSwitchReplicatedInfo(transactionId, replicationGroupId);
+
+        if (enabledColocationFeature) {
```

Comment:
Please check my comment in ZonePartitionReplicaListener.

Reply:
Which one? I only found this https://github.com/apache/ignite-3/pull/5241#discussion_r1962075163 but it doesn't seem to be applicable here